src/stat/fsext.rs

```diff
@@ -24,28 +26,30 @@ impl BirthTime for Metadata {
     fn pretty_birth(&self) -> String {
         self.created()
             .ok()
-            .and_then(|t| t.elapsed().ok())
-            .map(|e| pretty_time(e.as_secs() as i64, e.subsec_nanos() as i64))
-            .unwrap_or("-".to_owned())
+            .and_then(|t| t.duration_since(std::time::UNIX_EPOCH).ok())
```

Comment:
Import either `std::time` or `std::time::UNIX_EPOCH`.  I'm not really sure why this was using `elapsed()` before, so good catch.

Reply:
done